### ISSUE TITLE
geolocation

### ISSUE THREAD

**Issue by ktoddbrown** [ktoddbrown/CoSoDat#3]
Lat-long doesn't really do a great job preserving precision on their own. Maybe consider something like https://spatialparalysis.xyz/blog/dggs-eli5/ or https://s2geometry.io/devguide/s2cell_hierarchy.html

**Comment by dylanbeaudette** [comment]
That is nice, but can we also provide a "relative precision" field to describe a representative area via radius or similar?

**Comment by ktoddbrown** [comment]
What bothers me is the information loss here. Relative precision is certainly appropriate and an adequate solution for many use cases. But this doesn't accurate reflect the area encoded in the FIPS codes. Reference shapefiles would be one way to get around this but I think there could be a novel machine learning application that reflected regions instead of point precisions possible in the fugure.

**Comment by dylanbeaudette** [comment]
Agreed. I've been trying for years to get location "precision" added to all of our point data in NASIS / KSSL. There hasn't been much progress because of a lack of standards / methods. I'm all in favor of using a common (radius, landscape facet (?), hexagon, whatever) to ensure that point data aren't immediately interpreted as "points". We need a new hashtag "#regionsNotPoints".
